src/components/Cards/CardHeightMatching.js

```diff
@@ -45,6 +45,10 @@ class CardHeightMatching extends React.Component {
   }
 
   _matchHeights(selectors = this._selectors) {
+    if (!this._container) {
```

Comment:
just in case...it doesn't hurt...thanks!

Reply:
yeah, this was the source of the error @michaelkro mentioned before. Since _matchHeights is called inside of debounce, this can be called after the component has unmounted which will make _container null.  I didn't see a good way to cancel the debounced call, unfortunately.